### ISSUE TITLE
Free trial just vanished 

### ISSUE THREAD

**Issue by meepstertron** [cameronking4/sketch2app#5]
I just made a account using google and claimed my 3 days of a free trial and wanted to convert my sketch to code and I clicked on the list all options button and it showed me a scuffed "upgrade to use" 
![image](https://github.com/user-attachments/assets/248f7a04-a583-43f3-80b5-4243383c3541)

I reloaded the page and my 9999999 tokens (from the free trial for 3 days were gone? I'm not sure what I did wrong help aprechiated
![app Mockup](https://github.com/user-attachments/assets/d4780c2b-4a09-4aa6-8bc1-86873a77fac1)

**Comment by meepstertron** [comment]
I thought this would be something fun to do over the weekend to see if I like it 
![image](https://github.com/user-attachments/assets/c786032d-48db-4a88-922a-ac247e773ba5)

**Comment by cameronking4** [comment]
Yeah free trial is 24 hours. If you share the login email (gmail or GitHub) I can credit you some tokens to try it out more!Thanks,Cameron On Sep 27, 2024, at 2:06 PM, meepstertron ***@***.***> wrote:﻿Re: [cameronking4/sketch2app] Free trial just vanished (Issue #5)I thought this would be something fun to do over the weekend to see if I like itimage.png (view on web)—Reply to this email directly, view it on GitHub or unsubscribe.You are receiving this email because you are subscribed to this thread.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by PaOneAdk** [comment]
> Yeah free trial is 24 hours. If you share the login email (gmail or GitHub) I can credit you some tokens to try it out more!Thanks,Cameron On Sep 27, 2024, at 2:06 PM, meepstertron ***@***.***> wrote:﻿Re: [cameronking4/sketch2app] Free trial just vanished (Issue #5)I thought this would be something fun to do over the weekend to see if I like itimage.png (view on web)—Reply to this email directly, view it on GitHub or unsubscribe.You are receiving this email because you are subscribed to this thread.Triage notifications on the go with GitHub Mobile for iOS or Android.

Hello, can I also get more? I somehow lost my all tokens after around like less than 10 chat exchanges. I activated free trial just today morning. If yes, Where so I provide my email?
